modules/pel/peltool/peltool.py

```diff
@@ -307,6 +307,23 @@ def main():
 
         sys.exit(0)
 
+    if args.list:
+        if not os.path.isdir(args.list):
+            sys.exit(f"{args.list} is not a valid directory")
+        print("This function is under development")
+        final_summary={}
+        for root, _, files in os.walk(args.list):
+            for file in files:
+                if args.extension and args.extension != os.path.splitext(file)[1]:
+                    continue
+                eid, summary = parseAndSummarise(os.path.join(root, file), config, args.delete)
+                if(eid):
+                    final_summary[eid] = summary
```

Comment:
do we have guarantee that PEL id won't duplicate? or do we need to indicate some error message?